src/core/server/server.api.md

```diff
@@ -14,6 +14,8 @@ import { Observable } from 'rxjs';
 import { Request } from 'hapi';
 import { ResponseObject } from 'hapi';
 import { ResponseToolkit } from 'hapi';
+import { SavedObjectsNamespace as SavedObjectsNamespace_2 } from 'src/core/server';

```

Comment:
@skaapgif do you happen to know why I'm getting duplicate imports here? `SavedObjectsNamespace` is re-exported from `src/core/server`, but so is everything else from 'src/core/server/saved_objects`...

Reply:
This happens when `api-extractor` thinks these are different types that share the same name.

It seems like it goes away if you only import relatively inside of `src/core/server`

```
diff --git a/src/core/server/saved_objects/serialization/index.ts b/src/core/server/saved_objects/serialization/index.ts
index 0e755d1aba..ec379e78d9 100644
--- a/src/core/server/saved_objects/serialization/index.ts
+++ b/src/core/server/saved_objects/serialization/index.ts
@@ -25,7 +25,7 @@
 /* eslint-disable @typescript-eslint/camelcase */
 
 import uuid from 'uuid';
-import { SavedObjectsNamespace } from 'src/core/server/saved_objects';
+import { SavedObjectsNamespace } from '../';
 import { SavedObjectsSchema } from '../schema';
 import { decodeVersion, encodeVersion } from '../version';
 import {
diff --git a/src/core/server/saved_objects/service/saved_objects_client.ts b/src/core/server/saved_objects/service/saved_objects_client.ts
index e8fae3df33..e42998d120 100644
--- a/src/core/server/saved_objects/service/saved_objects_client.ts
+++ b/src/core/server/saved_objects/service/saved_objects_client.ts
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { SavedObjectsNamespace } from 'src/core/server';
+import { SavedObjectsNamespace } from './lib';
 import { SavedObjectsRepository } from './lib';
 
 import { SavedObjectsErrorHelpers } from './lib/errors';
```

Reply:
Ah that makes sense...I had done the absolute imports to make TypeScript happy with the module augmentation that we're doing in x-pack.

I might have to make the OSS SavedObjectsNamespace type be `string | undefined | symbol`, rather than augmenting it in x-pack then